### PR TITLE
Fixes #36328 - display short host name in Content Hosts page

### PR DESCRIPTION
**Pull request by lfu**
#### What are the changes introduced in this pull request?
Display host's FQDN or short name based on setting `display_fqdn_for_hosts` in `Content Hosts` page.

#### Considerations taken when implementing this change?
Depends on https://github.com/theforeman/foreman/pull/9613

#### What are the testing steps for this pull request?
Make sure https://github.com/theforeman/foreman/pull/9613 is available in your testing environment.
Verify the host name displayed in `Content Hosts` page is based on setting `display_fqdn_for_hosts`

### CODE REVIEW

**Comment by lfu** [comment]
[test katello]

**Review by jeremylenz — APPROVED** [review]
LGTM!

image

ACK :+1:

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-errata.html [hunk 1]
```diff
@@ -53,8 +53,8 @@ <h3 translate ng-show="selectedErrataOption === 'current'">Installable Errata</h
 
     <span data-block="list-actions" ng-hide="contentHost.readonly">
       <div bst-modal="applySelected()" model="host">
-        <div data-block="modal-header" translate>Apply Errata to Content Host "{{host.name}}"?</div>
-        <div data-block="modal-body" translate>Are you sure you want to apply Errata to content host "{{ host.name }}"?</div>
+        <div data-block="modal-header" translate>Apply Errata to Content Host "{{host.displayname}}"?</div>
+        <div data-block="modal-body" translate>Are you sure you want to apply Errata to content host "{{ host.displayname }}"?</div>
```
Comment (Gauravtalreja1) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-errata.html:57]: @lfu Do you think this should be {{host.display_name}}?

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-traces.html [hunk 2]
```diff
@@ -39,9 +39,9 @@ <h4 translate>Tracer helps administrators identify applications that need to be
   <div data-extend-template="layouts/partials/table.html">
     <span data-block="list-actions" ng-hide="contentHost.readonly">
       <div bst-modal="performViaRemoteExecution(false)" model="host">
-        <div data-block="modal-header" translate>Restart Services on Content Host "{{host.name}}"?</div>
+        <div data-block="modal-header" translate>Restart Services on Content Host "{{host.displayname}}"?</div>
         <div data-block="modal-body">
-          <span translate>Are you sure you want to restart services on content host "{{ host.name }}"?</span>
+          <span translate>Are you sure you want to restart services on content host "{{ host.displayname }}"?</span>
```
Comment (Gauravtalreja1) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-traces.html:44]: @lfu Same here

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-errata.html (hunk 1 above)
Comment (lfu) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-errata.html:57]: You are correct!

**Pull request merged**